app/src/main/kotlin/com/hippo/ehviewer/client/parser/ParserUtils.kt

```diff
@@ -41,6 +41,10 @@ object ParserUtils {
         return formatter.format(time.toLocalDateTime())
     }
 
+    fun parseDate(str: String): LocalDateTime {
+        return formatter.parse(str)
```

Comment:
nit: return `Long` to mirror the behaviour of `formatDate`.

Reply:
Agreed, timestamps are more widely used in this project.